app/components/ChallengesMapView.js

```diff
@@ -71,17 +70,17 @@ export default class ChallengesMapView extends Component {
 				<View style={{flex:1}}>
 					<MapView
 						style={styles.map}
-						loadingEnabled={true}
-						showUserLocation={true}
+						loadingEnabled={false}
```

Comment:
We don't want this set to `false`, we want it set to `true`. Setting this parameter shows a spinner but _only while the map is loading_. Once the map has loaded, the spinner goes away. Setting it to false allows you to create your own custom loading screens. Setting it to false without any loading screen however means the user will just see a blank screen while the map loads. If you are having issues with the map loading, talk to @yoojeong. She was able to get the map to load just fine on her phone and did not need to turn off the loading indicator.